app/utils/scheduling.py

```
@@ -9,6 +9,7 @@
 import psutil
 import threading
 import time
+import select
```

Ruff (0.11.9)

`select` imported but unused

Remove unused import: `select`

(F401)